core/tests/protractor_utils/AdminPage.js

```diff
@@ -268,6 +306,139 @@ var AdminPage = function() {
       expect(name).toEqual(expectedUsernamesArray[j]);
     }
   };
+
+  this.getMiscTab = async function() {
+    await miscTabButton.click();
+    await waitFor.pageToFullyLoad();
+  };
+
+  this.uploadTopicSimilarities =
+    async function(relativePathToSimilaritiesFile) {
+      absPath = path.resolve(__dirname, relativePathToSimilaritiesFile);
+      await waitFor.visibilityOf(chooseSimilarityFileInput,
+        'Similarity upload form is not visible');
+      await chooseSimilarityFileInput.sendKeys(absPath);
+      await waitFor.elementToBeClickable(similarityFileUploadButton,
+        'Upload button taking too long to be clickable');
+      await similarityFileUploadButton.click();
+    };
+
+  this.expectSimilaritiesToBeUploaded = async function() {
+    await waitFor.textToBePresentInElement(statusMessage,
+      'Topic similarities uploaded successfully.');
+    return true;
+  };
+
+  this.expectUploadError = async function() {
+    await statusMessage.getText().then((text) => {
+      if (text.match('Server')) {
+        return true;
+      }
+      return false;
+    });
+  };
+
+  this.downloadSimilarityFile = async function() {
+    await waitFor.elementToBeClickable(similarityDownloadButton,
+      'Download similarity file button not clickable');
+    await similarityDownloadButton.click();
+  };
+
+  this.expectFileToBeDownloaded = async function() {
+    var filename = global.downloadsPath + '/topic_similarities.csv';
+    browser.driver.wait(function() {
+      return fs.existsSync(filename);
+    }, 30000);
+  };
+
+  this.clearSearchIndex = async function() {
+    await waitFor.elementToBeClickable(searchIndexClearButton,
+      'Clear search index button not clickable');
+    await searchIndexClearButton.click();
+    await browser.switchTo().alert().accept();
+  };
+
+  this.expectSearchIndexToBeCleared = async function() {
+    await waitFor.textToBePresentInElement(statusMessage,
+      'Index successfully cleared.');
+    return true;
```

Comment:
Are there any lasting UI changes from clearing the search index so we can confirm that it worked? If so, that's what should go here, and this `waitFor` should go in `clearSearchIndex`

Reply:
I looked around the website on a local development server, but I couldn't find a place on the website where the search index could be accessed/seen on the UI (I made a bunch of queries on the 'library' page, but I couldn't find a place where my queries were indexed).

Do you have any ideas about where I might be able to find the search index?

Reply:
It might well be the case that there isn't any UI indication we can use here. Since you looked but couldn't find any, that's good enough for me (I don't know of any), so let's not worry about this